### PR TITLE
Lists updated instantly.

### PR DESCRIPTION
**Pull request by m-murad**
Fixes issue #171 

Please review.

### CODE REVIEW

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java [hunk 1]
```diff
@@ -34,7 +34,6 @@
 import android.view.animation.Animation;
 import android.view.animation.AnimationUtils;
 import android.widget.ImageButton;
-import android.widget.TextView;
```
Comment (free4murad) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java]: This was an unused import statement.

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java [hunk 2]
```diff
@@ -201,6 +208,8 @@ private void setUpFavTriggers() {
         else{
             showList();
         }
+
+        store.close();
```
Comment (m-murad) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: `store` was not closed earlier.
This was causing resource leaks.

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java [hunk 3]
```diff
@@ -35,8 +35,6 @@
...
-import android.widget.Toast;
```
Comment (m-murad) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java:39]: These were unused imports.

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java [hunk 4]
```diff
@@ -204,6 +211,8 @@ private void setUpFavTriggers() {
...
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: It is generally good to avoid mixing different works in PR :)

Comment (m-murad) [reply mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: @garg-anuj 
Sure
I will update the PR and send a new PR for resource leak.

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java [hunk 5]
```diff
@@ -100,6 +100,13 @@ public void onActivityCreated(Bundle state) {
         setUpTriggers();
     }
 
+    @Override public void onResume() {
+        super.onResume();
+        if( getArguments().getString("state").equalsIgnoreCase("From Favourite")) {
+            setUpFavTriggers();
+        }
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: corresponding else case is missing

**Review by KeenWarrior — CHANGES_REQUESTED** [review]

**Comment by m-murad** [comment]
@garg-anuj 
Done

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java [hunk 6]
```diff
@@ -100,6 +100,15 @@ public void onActivityCreated(Bundle state) {
         setUpTriggers();
     }
 
+    @Override public void onResume() {
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: @free4murad Why you put `@Override` in same line of method everytime :laughing:

Comment (m-murad) [reply mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: Sorry I think there is some issue with Android studio on my laptop.
I will make sure that it doesn't happen again

**Review by KeenWarrior — APPROVED** [review]

**Pull request merged**